pom.xml

```diff
@@ -0,0 +1,291 @@
+                <version>${powermock.version}</version>
+                <scope>test</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.wso2.carbon.extension.identity.authenticator.utils</groupId>
+                <artifactId>org.wso2.carbon.extension.identity.helper</artifactId>
+                <version>${identity.extension.utils}</version>
+                <scope>provided</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.wso2.carbon.identity.outbound.auth.oidc</groupId>
+                <artifactId>org.wso2.carbon.identity.application.authenticator.oidc</artifactId>
+                <version>${org.wso2.carbon.identity.outbound.auth.oidc}</version>
+                <scope>provided</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <repositories>
+        <repository>
+            <id>wso2-nexus</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/groups/wso2-public/</url>
+            <releases>
+                <enabled>true</enabled>
+                <updatePolicy>daily</updatePolicy>
+                <checksumPolicy>ignore</checksumPolicy>
+            </releases>
+        </repository>
+        <repository>
+            <id>wso2.releases</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/releases/</url>
+            <releases>
+                <enabled>true</enabled>
+                <updatePolicy>daily</updatePolicy>
+                <checksumPolicy>ignore</checksumPolicy>
+            </releases>
+        </repository>
+        <repository>
+            <id>wso2.snapshots</id>
+            <name>Apache Snapshot Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/snapshots/</url>
+            <snapshots>
+                <enabled>true</enabled>
+                <updatePolicy>daily</updatePolicy>
+            </snapshots>
+            <releases>
+                <enabled>false</enabled>
+            </releases>
+        </repository>
+    </repositories>
+    <properties>
+        <maven.buildnumber.plugin.version>1.4</maven.buildnumber.plugin.version>
+        <identity.framework.version>5.12.387</identity.framework.version>
+        <identity.framework.imp.pkg.version.range>[5.12.387, 6.0.0)</identity.framework.imp.pkg.version.range>
+        <carbon.identity.package.export.version>${project.version}</carbon.identity.package.export.version>
+        <jacoco.version>0.8.2</jacoco.version>
+
+        <carbon.identity.version>[5.14.0, 6.0.0]</carbon.identity.version>
+        <commons.logging.version>1.1.1</commons.logging.version>
+        <carbon.kernel.version>4.4.3</carbon.kernel.version>
+        <oltu.version>1.0.0.wso2v2</oltu.version>
+        <org.apache.oltu.oauth2.client.version>1.0.0</org.apache.oltu.oauth2.client.version>
```

Comment:
Remove unused property. Check for other places as well.

Reply:
done